lib/build-query/queries/worksSubjectOfWork.js

```diff
@@ -0,0 +1,9 @@
+// Find works that are the subject of another work.
+const worksSubjectOfWork = (workId) => ({
+  _scope: 'work',
+  subjecOfWork: {
```

Comment:
typo: the field is called `subjectOfWork` - here it says `subjecOfWork` without the `t`